litellm/tests/test_bedrock_completion.py

```diff
@@ -1,33 +1,33 @@
 # @pytest.mark.skip(reason="AWS Suspended Account")
-# import sys, os
```

Comment:
My team's (litellm) account on AWS has been suspended which is why this test is commented out. Until we get access to our AWS account again

Reply:
@ishaan-jaff no worries, commented them out now. I'd appreciate a quick merge by next Monday if possible since my team is eager to test out all these new Mistral models next week. Open to make any changes required though.

Reply:
These tests will be re-added as soon as AWS give us access to our own account

Reply:
@ti3x yes, will have this ready for by Monday